root/etc/backup-data.hooks/backup-notify

```diff
@@ -33,8 +33,12 @@ else
    subject="Backup $backup [$hostname]:"
 fi
 
+notify_from=$(/sbin/e-smith/config getprop root SenderAddress)
+if [ -z $notify_from ]; then
+    notify_from=$(/sbin/e-smith/db $db getprop $backup NotifyFrom)
```

Comment:
Shall we emit a deprecation warning here?

Reply:
I can't output anything here, bu could log something inside `/var/log/messages`.
Still I'm not sure is useful: nobody read the warnings as nobody read the documentation :)